app/controllers/submit.js

```diff
@@ -333,8 +338,38 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             this.send('next', this.get('_names.0'));
         },
         resetFileUpload() {
-            this.send('changeState', State.START);
-            this.set('fileAndNodeLocked', false);
+            var promisesArray = [];
+            var filePromises = [];
+            this.get('filesUploadedForPreprint').forEach((file) => {
+                filePromises.push(this.get('fileManager').deleteFile(file).then(() => {
+                    this.get('toast').info('File removed!');
+                }).catch(() => {
+                    this.get('toast').error('Could not remove uploaded file');
+                }));
+            });
+
+            Ember.RSVP.all(filePromises).then(() => {
+                this.get('projectsCreatedForPreprint').forEach((project) => {
+                    promisesArray.push(this.store.adapterFor('node').ajax(project.get('links.self'), 'DELETE').then(() => {
```

Comment:
I don't think node.deleteRecord() works?  The delete record on the nodeActions mixin finds the node that has already been loaded on the route.  So I temporarily did this until we can add functionality to ember0sf.